web/docs/get-started.md

```diff
@@ -205,6 +205,14 @@ storage.
   </TabItem>
 </Tabs>
 
+:::caution Unsafe Pipelines
+Some operators are inherently unsafe because of their side effects, e.g.,
+reading a file, acquiring packets from the network, or using a third-party
+library. We do not allow these operators by default in managed pipelines. You
```

Comment:
What do you mean with "or using a third-party library?" The restriction only applies to operators that are marked _local_ internally. For example, we do allow interacting with third-party libraries like Apache Parquet or simdjson in operators that can safely run in any location.

Reply:
> What do you mean with "or using a third-party library?"

Something that's not a native operator implementation. An example would be libpcap, although that's also already covered by the previous point. Basically anything that a user can controls with one level of indirection.

> The restriction only applies to operators that are marked local internally. For example, we do allow interacting with third-party libraries like Apache Parquet or simdjson in operators that can safely run in any location.

I don't think the concept of "local" is relevant here. Managed pipelines often run all of their operators on a single node. The user-facing option has `unsafe` in the name, so you need to come from a perspective of security in the argumentation.